Cron/CancelAbandonedOrders.php

```
@@ -48,6 +53,18 @@ public function execute(): void
             ->modify("-{$minutes} minutes")
             ->format('Y-m-d H:i:s');
 
+        $cancelStatus = $this->scopeConfig->getValue(
+            self::XML_CANCEL_STATUS,
+            ScopeInterface::SCOPE_STORE
+        );
+
+        $validCancelStatuses = $this->orderConfig->getStatuses();
+
+        if (!$cancelStatus || !isset($validCancelStatuses[$cancelStatus])) {
+            $this->logger->warning('Configured cancel status does not exist: ' . $cancel_status);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$cancel_status'. (undefined)

(UnusedLocalVariable)